django_connexion/apis/django_api.py

```diff
@@ -95,7 +95,7 @@ def get_request(self, request, *args, **params):
             request.method,
             headers=request.headers,
             form=request.POST,
-            query=request.GET,
+            query=django_utils.parse_query_params(request.GET),
```

Comment:
you don't need a custom function for that, just use `dict`

```suggestion
            query=dict(request.GET),
```

Reply:
true!